dive-spots.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Places to Scuba Dive</title>
+  </head>
+  <body>
+    <h1>Dive Spots</h1>
+    <p>There are many great spots to scuba dive. If it's a source of water; mostly likely, you will be able to submerge yourself under meters of water.</p>
```

Comment:
You might consider splitting this line up for legibility even though it doesn't have any child tags. 
`
<p>
  text
</p>
`
1: it's kind of long to be just one line
2: if in the future you do decide to put child elements in there, it'll already be formatted for legibility.